### ISSUE TITLE
Clean up link stragglers to old doc portal

### ISSUE THREAD

**Issue by r0ckarong** [SUSE/doc-caasp#725]
From email by @sknorr:

```
en-us/suse-caasp/4.0/html/caasp-deployment/_deployment_instructions.html:96: https://www.suse.com/documentation/sle-ha-15/index.html
en-us/suse-caasp/4.0/html/caasp-deployment/_deployment_instructions.html:174: https://www.suse.com/documentation/sle-ha-15/index.html
en-us/suse-caasp/4.0/html/caasp-deployment/_deployment_instructions.html:468: https://www.suse.com/documentation/sles-15/singlehtml/book_autoyast/book_autoyast.html
en-us/suse-caasp/4.0/html/caasp-deployment/_deployment_instructions.html:473: https://www.suse.com/documentation/sles-15/singlehtml/book_autoyast/book_autoyast.html
en-us/suse-caasp/4.0/html/caasp-deployment/_deployment_instructions.html:496: https://www.suse.com/documentation/sles-15/singlehtml/book_rmt/book_rmt.html#cha.rmt_client
en-us/suse-caasp/4.0/html/caasp-deployment/_about_this_guide.html:33: http://www.suse.com/documentation/
en-us/suse-caasp/4.0/html/caasp-deployment/_about_this_guide.html:38: http://www.suse.com/documentation/feedback.html
en-us/suse-caasp/4.0/html/caasp-admin/_troubleshooting.html:39: https://www.suse.com/documentation/sles-15/book_sle_admin/data/cha_adm_support.html
en-us/suse-caasp/4.0/html/caasp-admin/_about_this_guide.html:33: http://www.suse.com/documentation/
en-us/suse-caasp/4.0/html/caasp-admin/_about_this_guide.html:38: http://www.suse.com/documentation/feedback.html
en-us/suse-caasp/4.0/html/caasp-admin/_integration.html:34: https://www.suse.com/documentation/suse-enterprise-storage/
en-us/suse-caasp/4.0/html/caasp-admin/_integration.html:331: https://www.suse.com/documentation/cloud-application-platform-1/book-cap-guides/data/book-cap-guides.html
en-us/suse-caasp/4.0/html/caasp-quickstart/deployment.preparations.html:93: https://www.suse.com/documentation/sle-ha-15/index.html
en-us/suse-caasp/4.0/html/caasp-quickstart/deployment.preparations.html:171: https://www.suse.com/documentation/sle-ha-15/index.html
```

**Comment by sknorr** [comment]
Ftr, all of these links appear to be fixed in the 4.1 version -- this should just be a matter of some backports.

**Comment by r0ckarong** [comment]
> Ftr, all of these links appear to be fixed in the 4.1 version -- this should just be a matter of some backports.

@sknorr I'm explicitly told "no backports" to older versions unless they are significant product breaking bugs. I will close this issue.
